SWLOR.Game.Server/Feature/AbilityDefinition/Force/ThrowLightsaberAbilityDefinition.cs

```diff
@@ -133,7 +133,6 @@ private static void ThrowLightsaber1(AbilityBuilder builder)
                 .RequirementFP(2)
                 .IsCastedAbility()
                 .IsHostileAbility()
-                .UsesAnimation(Animation.LoopingConjure1)
```

Comment:
Dropped you a line in Discord - but did I cause this with a Master merge, or did you intentionally remove these?